ctc_lib/ctc_lib.go

```diff
@@ -19,19 +19,29 @@ package ctc_lib
 // This file declares all the package level globals
 
 import (
-	"fmt"
+	"bytes"
 	"os"
+	"path/filepath"
 
+	"github.com/GoogleCloudPlatform/runtimes-common/ctc_lib/config"
+	"github.com/GoogleCloudPlatform/runtimes-common/ctc_lib/constants"
 	"github.com/GoogleCloudPlatform/runtimes-common/ctc_lib/flags"
+	"github.com/GoogleCloudPlatform/runtimes-common/ctc_lib/logging"
 	log "github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
 )
 
 var exitOnError = true
 var Version string
-var emptyTemplate = "{{.}}"
-var defaultLogLevel = "info"
+var ConfigFile string
+var ConfigType = constants.ConfigType
```

Comment:
do you need this?

Reply:
yes its specifies the viper Config File. An example of it is in Demo.
```
func Execute() {
 	ctc_lib.Version = "1.0.1"	
+	ctc_lib.ConfigFile = "demo/ctc/testConfig.json"
 	ctc_lib.Execute(RootCommand)
 }
```

Reply:
The configType is set to Json by default, but you can also choose yaml or other supported types by setting this variable.